packages/mobile/src/navigator/types.tsx

```diff
@@ -123,7 +123,8 @@ export type StackParamList = {
   [Screens.TabNavigator]: {}
   [Screens.TransactionReview]: {
     reviewProps: ReviewProps
-    confirmationProps: TransferConfirmationCardProps | ExchangeConfirmationCardProps
+    transferConfirmationProps?: TransferConfirmationCardProps
```

Comment:
Is there a reason to change these? This changes from "one of these two is required" to "one, both, or neither of these two" will exist

Reply:
The reason why I did was that these two types are drastically different. Wasn't a problem when all we were doing was passing them through but I ran into issues when I needed to operate on them.

Is there an easy way to do conditionals based on types if we have one property with two potential types? For example, I was trying unsuccessfuly to do `if (typeof confirmationProps === TransferConfirmationCardProps`

Reply:
I think you're probably looking for a type guard: https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards

Reply:
Indeed I am! Will revert these changes and use this